plugins/aws-appmesh/config/netconfig.go

```diff
@@ -92,6 +97,27 @@ func New(args *cniSkel.CmdArgs) (*NetConfig, error) {
 		EnableIPv6:         config.EnableIPv6,
 	}
 
+	if config.PrevResult != nil {
```

Comment:
It'd be awesome if you could add a unit test for this unmarshalling logic.

Reply:
I will do that after our current milestone.